python/packages/azurefunctions/agent_framework_azurefunctions/_orchestration.py

```diff
@@ -96,23 +109,24 @@ def run(
         Args:
             messages: The message(s) to send to the agent
             thread: Optional agent thread for conversation context
-            **kwargs: Additional arguments (enable_tool_calls, response_format, etc.)
+            response_format: Optional Pydantic model for response parsing
+            **kwargs: Additional arguments (enable_tool_calls)
 
         Returns:
-            Task that will resolve to the agent response
+            Yield a task that will resolve to the agent response
 
         Example:
             @app.orchestration_trigger(context_name="context")
             def my_orchestration(context):
                 agent = app.get_agent(context, "MyAgent")
                 thread = agent.get_new_thread()
-                result = yield agent.run("Hello", thread=thread)
+                result = yield from agent.run("Hello", thread=thread)
         """
         message_str = self._normalize_messages(messages)
+        logger.debug(f"[DurableAIAgent] Running agent '{self.agent_name}' with message: {message_str[:100]}...")
 
         # Extract optional parameters from kwargs
         enable_tool_calls = kwargs.get("enable_tool_calls", True)
```

Comment:
Unrelated to your change but we should make this as an explicit keyword argument instead of pulling it from kwargs. You can either change it here or lets create a follow up task.